core/client/fs/src/main/java/alluxio/client/block/stream/BlockInStream.java

```diff
@@ -328,6 +339,13 @@ public WorkerNetAddress location() {
 
   @Override
   public boolean isLocal() {
-    return mLocal;
+    return CommonUtils.isLocalHost(mAddress);
```

Comment:
In that case we are overloading the term `local` which can lead to some confusion. Is there any way around this?